lib/galaxy/celery/tasks.py

```diff
@@ -332,10 +332,15 @@ def export_history(
 def prepare_dataset_collection_download(
     request: PrepareDatasetCollectionDownload,
     collection_manager: DatasetCollectionManager,
+    session: galaxy_scoped_session,
     task_user_id: Optional[int] = None,
 ):
+    if task_user_id:
+        user = session.query(model.User).get(task_user_id)
```

Comment:
```suggestion
        user = session.get(model.User, task_user_id)
```
we are moving to modern SQLAlchemy syntax (`session.get`  added in version 1.4)